### PR TITLE
Add new rule accounts_authorized_local_users to profile sap on ol7

### PR DESCRIPTION
**Pull request by xiruiyang**
modified:   ol7/profiles/sap.profile
	new file:   shared/checks/oval/accounts_authorized_local_users.xml
	new file:   shared/fixes/bash/accounts_authorized_local_users.sh
	new file:   shared/guide/system/software/sap/accounts_authorized_local_users.rule
	new file:   shared/guide/system/software/sap/var_accounts_authorized_local_users.var

#### Description:

- This is a replacement of the PR #2862

### CODE REVIEW

**Comment by xiruiyang** [comment]
@redhatrises Could you please take a review and merge the PR? Thanks.

File: ol7/profiles/sap.profile [hunk 1]
```diff
@@ -15,3 +15,5 @@ selections:
     - no_rsh_trust_files
     - package_ypbind_removed
     - package_ypserv_removed
+    - var_accounts_authorized_local_users=ol7forsap
```
Comment (matejak) [inline ol7/profiles/sap.profile]: As the `var_accounts_authorized_local_users` contains a regular expression, could you please reflect this in the variable name by adding the `_regex` suffix? We already follow that pattern e.g. in `shared/checks/oval/rsyslog_files_ownership.xml`.

**Comment by matejak** [comment]
Also please disregard suggestions from the Codacy service, which comes with some false positives as it is not fully set up yet.

**Comment by matejak** [comment]
Great, thank you very much for your work!

**Comment by mpreisler** [comment]
@matejak somehow you managed to merge this without it passing jenkins, now the build is broken on master. I am really curious how it happened because we have rules in place that should prevent this :-/

**Comment by mpreisler** [comment]
@openscap-ci test this please

**Comment by xiruiyang** [comment]
@mpreisler , the build was successful. It's only the codacy check that failed.  The same is for PR#2884.

**Comment by mpreisler** [comment]
> @mpreisler , the build was successful. It's only the codacy check that failed. The same is for PR#2884.

Yeah, I got confused by all the checks. I am used to the little icon meaning jenkins passed or failed, now it has a different meaning. Sorry about the confusion.

**Pull request merged**